features/user-profile.md

```diff
@@ -0,0 +1,226 @@
+# API Design Overview
+
+This document introduces the new Skygear User Record (Profile), which is
+re-designed from user record and user object, utilizing user profile discovery.
+The design depends on [the field-based ACL](field-based-acl.md) feature.
+
+## New User Record
+
+The newly designed user record contains different information of user.By
+setting field-based ACL, user record can serve as both public profile and
+private profile.
```

Comment:
this should read “user record can contain public attributes and private attributes”